macros/src/actix.rs

```diff
@@ -1057,3 +1064,37 @@ impl SerdeProps {
         props
     }
 }
+
+/// Supported flattening of embedded struct (https://serde.rs/variant-attrs.html).
+struct SerdeFlatten;
+
+impl SerdeFlatten {
+    /// Traverses the field attributes and returns the renamed value from the first matching
+    /// `#[serde(rename = "...")]` pattern.
+    fn from_field_attrs(field_attrs: &[Attribute]) -> bool {
```

Comment:
Let's rename this to `exists` because it returns a boolean.

Reply:
good point!